nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,60 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem
+*********
+
+Core library
+============
+
+* Added:
+
+  * New header :c:file:`nrf_modem_toolchain.h` for compiler attributes used in other header files.
+  * Binaries for the nRF9230 SoC for internal development.
+  * New header :c:file:`nrf_modem_os_rpc.h` for nRF9230 SoC RPC OS glue.
+
+AT interface
+============
+
+* Added:
+
+  * Logging for AT responses and notifications to the logging binaries.
+  * Format helpers to AT functions that takes a variable number of arguments to improve type checking.
+
+Sockets
+=======
+
+* Added:
+
+  * The capability to read the value of the :c:macro:`NRF_SO_KEEPOPEN` socket option with modem firmware version 2.0.2 and newer.
+  * The new :c:macro:`NRF_SO_IPV6_DELAYED_ADDR_REFRESH` socket option for delaying IPv6 address refresh until next time the device wakes up from PSM or eDRX sleep.
+    This will prevent the device from waking up solely to refresh the address.
+    Supported by modem firmware version 1.3.7 and newer and modem firmware version 2.0.2 and newer.
+
+* Removed:
+
+  * The ``nrf_sec_cipher_t``, ``nrf_sec_peer_verify_t``, ``nrf_sec_role_t``, and ``nrf_sec_session_cache_t`` types that were deprecated in version 2.6.0.
+  * The RAI socket options :c:macro:`NRF_SO_RAI_NO_DATA`, :c:macro:`NRF_SO_RAI_LAST`, :c:macro:`NRF_SO_RAI_ONE_RESP`, :c:macro:`NRF_SO_RAI_ONGOING`, and :c:macro:`NRF_SO_RAI_WAIT_MORE` that were deprecated in version 2.6.0.
+
+* The :c:func:`nrf_getaddrinfo()` function now supports up to three DNS requests in parallel.
```

Comment:
Nit: aren't they technically concurrent rather than parallel? As far as I remember they are still serialized on the modem side so they are never exactly parallel.